ui/lib/core/addon/components/form-field.js

```diff
@@ -114,7 +114,7 @@ export default class FormFieldComponent extends Component {
   }
   @action
   setAndBroadcastTtl(value) {
-    const alwaysSendValue = this.valuePath === 'expiry' || this.valuePath === 'safetyBuffer';
+    const alwaysSendValue = this.valuePath === 'safetyBuffer';
```

Comment:
I did a global search and this was the only model with an `attr` of `expiry` so it appears that this conditional was only for pki config